build/run-tests.sh

```diff
@@ -3,7 +3,7 @@ set -e
 
 SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
 
-function stop_containers() {
+stop_containers() {
```

Comment:
Was this actually a required change?

Reply:
The tests were breaking while running the `build/run-tests.sh` script. I could have worked around it but it seemed like fixing it was the best way forward. The shabang in the file specifies `sh` as the executor but the `function` keyword is ksh/bash specific syntax. As this isn't required as part of the Splunk logging change, I'm happy to remove it from this PR.
 
![Screenshot from 2020-08-13 09-04-14](https://user-images.githubusercontent.com/17098249/90109676-0fc05900-dd44-11ea-8364-f16c240eed6d.png)